include/mbedtls/config.h

```diff
@@ -88,6 +88,16 @@
  */
 //#define MBEDTLS_NO_UDBL_DIVISION
 
+/**
+ * \def MBEDTLS_SIZE_OPTIMIZED_MULADDC
```

Comment:
Our naming convention would be `MBEDTLS_BIGNUM_xxx`. Rather than make this specifically about `MULADDC`, I think it should be about optimizing for code size vs performance. There's already a `MBEDTLS_SHA256_SMALLER`, so I think a good name would be `MBEDTLS_BIGNUM_SMALLER`.